adsenrich/utils.py

```diff
@@ -23,22 +26,26 @@ def u2asc(input):
 
     if not isinstance(input, test_type):
         try:
-            input = input.decode('utf-8')
+            input = input.decode("utf-8")
         except UnicodeDecodeError:
-            raise UnicodeHandlerError('Input must be either unicode or encoded in utf8.')
+            raise UnicodeHandlerError("Input must be either unicode or encoded in utf8.")
 
     try:
         output = unidecode.unidecode(input)
     except UnicodeDecodeError:
-        raise UnicodeHandlerError('Transliteration failed, check input.')
+        raise UnicodeHandlerError("Transliteration failed, check input.")
 
     if not isinstance(input, test_type):
-        output = output.encode('utf-8')
+        output = output.encode("utf-8")
 
     return output
 
 
-def issn2bib(token=None, url=None, issn=None):
+def issn2info(token=None, url=None, issn=None, return_info="bibstem"):
```

Comment:
@seasidesparrow I changed this from issn2bib to issn2info to be more generic and have the capability of returning either the bibcode or the publisher. Is this okay? Will this break things elsewhere?